### PR TITLE
Add guide for Helm support

### PR DESCRIPTION
**Pull request by Shubbhang351**
1. **What this PR does / why we need it**:

Add guide for Helm support

 [Helm Guide](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/vshubhang/helm-guid/docs/guides/tanzu-mission-control_helm.md)

### CODE REVIEW

File: docs/guides/tanzu-mission-control_helm.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
+---
+Title: "Helm  Resource"
+Description: |-
+    Adding Helm resource to a cluster.
+---
+
+# Defining Helm
+
+Enable the Helm service when you want to install Helm charts on a cluster or cluster group.
+When you enable the Helm service on a cluster or cluster group, you can then create releases in your cluster from Helm charts stored in the Bitnami repository.
```
Comment (ishangupta-ds) [inline docs/guides/tanzu-mission-control_helm.md:10]:
```suggestion
When you enable the Helm service on a cluster or cluster group, you can then create releases in your cluster from Helm charts stored in the Bitnami Helm repository.
```

File: docs/guides/tanzu-mission-control_helm.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+[helm-feature]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-0927CDC8-A5C1-4FAE-9A7C-8A5D62FDF8D8.html
+
+
+## Helm Release
+
+The `tanzu-mission-control_helm_release` resource allows you to install, update, get and delete helm chart to a particular scope through Tanzu Mission Control.
+
+Before creating helm charts user needs to enable the helm service on a particular scope cluster or cluster group and for enable user can use `tanzu-mission-control_helm_feature` resource.
+The`feature_ref` field of `tanzu-mission-control_helm_release` when specified, ensures clean up of this Terraform resource from the state file by creating a dependency on the Helm feature when the Helm feature is disabled.
+To add a helm charts, you must be associated with the cluster.admin or clustergroup.admin role.
```
Comment (ishangupta-ds) [inline docs/guides/tanzu-mission-control_helm.md:31]:
```suggestion
To create a Helm release, you must be associated with the cluster.admin or clustergroup.admin role.
```

File: docs/guides/tanzu-mission-control_helm.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+# Defining Helm
+
+Enable the Helm service when you want to install Helm charts on a cluster or cluster group.
+When you enable the Helm service on a cluster or cluster group, you can then create releases in your cluster from Helm charts stored in the Bitnami repository.
+
+# Terraform Resources for Helm
+
+## Helm Feature
+
+The `tanzu-mission-control_helm_feature` resource allows you to enable and disable helm feature to a particular scope through Tanzu Mission Control.
```
Comment (ishangupta-ds) [inline docs/guides/tanzu-mission-control_helm.md:16]:
```suggestion
The `tanzu-mission-control_helm_feature` resource allows you to enable and disable helm feature for a particular scope through Tanzu Mission Control.
```

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: docs/guides/tanzu-mission-control_helm.md (hunk 1 above)
Comment (Shubbhang351) [reply docs/guides/tanzu-mission-control_helm.md]: Updated PTAL

File: docs/guides/tanzu-mission-control_helm.md (hunk 2 above)
Comment (Shubbhang351) [reply docs/guides/tanzu-mission-control_helm.md]: Updated PTAL

File: docs/guides/tanzu-mission-control_helm.md (hunk 3 above)
Comment (Shubbhang351) [reply docs/guides/tanzu-mission-control_helm.md]: Updated PTAL

**Review by ishangupta-ds — APPROVED** [review]

**Pull request merged**